analysed-packages/glib-networking/version-2.72.2/glib-networking-2.72.2-SPDX2TV.spdx

```diff
@@ -0,0 +1,6468 @@
+FileCopyrightText: <text> Copyright (C) 2011 glib-networking's Free Software Foundation, Inc. Nguyễn Thái Ngọc Duy <pclouds@gmail.com>, 2011-2013. </text>
+
+##File
+
+FileName: glib-networking-2.72.2.tar.xz/glib-networking-2.72.2/po/es.po
+SPDXID: SPDXRef-item544396
+FileChecksum: SHA1: 1bb97fca7a735223bd1ae5549ed4482a3994c805
+FileChecksum: SHA256: 35d5ec88d67eddf377dd9364d32481621d173d1e1f38bd4469223b074b3be87f
+FileChecksum: MD5: 6d3259eb3a6584a3add7448a4664d93d
+LicenseConcluded: LicenseRef-LGPL-2.1-or-later
+LicenseComments: <text> The information in the file is:
+
+This file is distributed under the same license as the glib-networking package.
+
+Thus, we concluded LGPL-2.1-or-later. </text>
+LicenseInfoInFile: LicenseRef-Same-license-as
+LicenseInfoInFile: LicenseRef-free-unknown
+FileCopyrightText: <text> Copyright (C) 2011 glib-networking's COPYRIGHT HOLDER  Jorge González <jorgegonz@svn.gnome.org>, 2011. Daniel Mustieles <daniel.mustieles@gmail.com>, 2011-2022. </text>
+
+##File
+
+FileName: glib-networking-2.72.2.tar.xz/glib-networking-2.72.2/tls/tests/files/update-test-database.py
+SPDXID: SPDXRef-item544317
+FileChecksum: SHA1: 04e1aa7f0f57de7b79f0ec009d22dca626591551
+FileChecksum: SHA256: 9867f766480226e1ea34657db152b32cfcb432af2da515c6a8c82c79747595c4
+FileChecksum: MD5: 77fc8cd2bf89010745e925da5cd8c2db
+LicenseConcluded: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-lgpl-2.1-plus
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: glib-networking-2.72.2.tar.xz/glib-networking-2.72.2/tls/tests/files/update-chain-with-new-root.py
+SPDXID: SPDXRef-item544284
+FileChecksum: SHA1: fa751f48ae765c25e90dcd1105cc1a46bbaf323e
+FileChecksum: SHA256: 5c2c58d5d425aeb40edde3ba9563ed6e962addaf446ebb1c56f258a134de6246
+FileChecksum: MD5: 739d7a8fa90232c0a247ca12675f3f40
+LicenseConcluded: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-lgpl-2.1-plus
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: glib-networking-2.72.2.tar.xz/glib-networking-2.72.2/tls/tests/files/update-certificate-test.py
+SPDXID: SPDXRef-item544261
+FileChecksum: SHA1: aab3ed978de6571ea35512f87d5297d0c2a82489
+FileChecksum: SHA256: 45d3c67704ff396bc8b1a9dcf36465a254d9be05f1cb413b251b2046f06e3788
+FileChecksum: MD5: c116a9f333311164f5fa29752792c213
+LicenseConcluded: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-lgpl-2.1-plus
+FileCopyrightText: NOASSERTION
+
+##File
+
+FileName: glib-networking-2.72.2.tar.xz/glib-networking-2.72.2/tls/tests/pkcs11/pkcs11t.h
+SPDXID: SPDXRef-item544323
+FileChecksum: SHA1: e29bb7cbf0eac413f7a870fa6b1d2b06e7a72287
+FileChecksum: SHA256: 5824e4b4b8d9078c85f27b0616342c09ce1be084bf0f8e3818f08353d855ad17
+FileChecksum: MD5: 993d51c4201f279e3bf6d640de3d3cc7
+LicenseConcluded: LicenseRef-MPL-2.0 AND LicenseRef-RSA-Cryptoki
```

Comment:
The RSA-Cryptoki license requires the following acknowledgements, which are missing:

This software contains RSA Security Inc. PKCS  11 Cryptographic Token Interface (Cryptoki).

This software contains material derived from the RSA Security Inc. PKCS  11 Cryptographic Token Interface (Cryptoki).